completions/brctl

```diff
@@ -1,4 +1,19 @@
 # bash completion for brctl                                -*- shell-script -*-
+_brctl_interfaces()
+{
+    local PATH=$PATH:/sbin
+    COMPREPLY=($({
+        if [[ -z "$1" ]]; then
```

Comment:
With `brctl show $1` splitting and unintentional globbing can happen. With `[[ -z $1 ]]` it can't, shfmt is correct in doing the cleanup.